imageroot/actions/configure-module/validate-input.json

```
@@ -1,8 +1,52 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "configure-module input",
-    "$id": "http://schema.nethserver.org/crowdsec/configure-module-input.json",
+    "title": "Configure crowdsec",
+    "$id": "http://nethserver.org/json-schema/task/input/crowdsec/configure-module",
     "description": "Configure crowdsec",
-    "examples": [],
-    "type": "object"
+    "examples": [
+        {
+            "helo_host": "foo.domain.com",
+            "receiver_emails": "user@domain.com,user@domain.org",
```

Can we use and array?
It's simpler to use and also to validate inside the UI.
```suggestion
            "receiver_emails": ["user@domain.com", "user@domain.org"],
```